content/en/04-tutorials/03-integrations/03-otel.md

```diff
@@ -1,22 +1,41 @@
 ---
-title: "Export Data to OpenTelemetry"
-metaTitle: "Tutorials | Integrations and Alerts | Pixie <> OpenTelemetry"
-metaDescription: "Export Data to OpenTelemetry"
+title: "Export OpenTelemetry Data"
+metaTitle: "Tutorials | Integrations and Alerts | Export OpenTelemetry Data"
+metaDescription: "Export OpenTelemetry Data"
 order: 3
 redirect_from:
     - /tutorials/otel/
 ---
 
+This tutorial will demonstrate how to export Pixie data in the OpenTelemetry (OTel) format.
 
-Pixie comes packaged with an OpenTelemetry exporter. You can write PxL scripts that define the transformation of Pixie DataFrames into OpenTelemetry data. This article walks through a script that exports HTTP data collected by Pixie into an OpenTelemetry endpoint. More detailed PxL documentation for the OpenTelemetry integration is available [here](/reference/pxl/otel-export). 
+To do this, we will take the following steps:
 
+1. Deploy an OTel collector to our cluster.
+2. Write a PxL script that uses the [OTel methods](/reference/pxl/otel-export/) to transformation Pixie DataFrames into OTel data.
+3. Setup the Pixie [Plugin System](/reference/plugins/plugin-system/) to run the PxL script at regularly scheduled intervals.
 
-## Example OpenTelemetry Export PxL Script
+Note that [Pixie's API](/using-pixie/api-quick-start/) can also be used to run the PxL scripts developed in Step 2, however this tutorial will cover the Plugin System only.
 
-The following [PxL script](/tutorials/pxl-scripts/write-pxl-scripts/#overview) calculates the rate of HTTP requests made to each pod in your cluster and exports that data as an OpenTelemetry Gauge metric.  
+## Deploy the OTel Collector
 
+If you don't already have an OTel collector set up, you can follow the directions to deploy our demo collector [here](https://github.com/pixie-io/pixie-demos/tree/main/otel-collector).
 
-```python
+Most OTel collectors are configured to forward metrics to Prometheus or Jaeger, but our demo collector simply outputs the metrics it receives to the logs.
```

Comment:
Most OTel collectors are configured to forward metrics to _other tools such as_ Prometheus or Jaeger.